src/jquery.ime.js

```diff
@@ -601,7 +602,7 @@
 		if ( !range || !range.collapsed || range.startContainer.nodeType !== Node.TEXT_NODE ) {
 			return '';
 		}
-		return range.startContainer.nodeValue.substring(
+		return range.startContainer.nodeValue.slice(
```

Comment:
Automatic change by eslint, which requires slice() instead of substring() and substr().